packages/@ngtools/webpack/src/index.ts

```diff
@@ -5,7 +5,7 @@ import { satisfies } from 'semver';
 // but can't until we split global and local packages.
 // See https://github.com/angular/angular-cli/issues/8107#issuecomment-338185872
 try {
-  const version = require('typescript').version;
+  const version = require('typescript').version.match(/^[0-9]+\.[0-9]+\.[0-9]+/)[0];
   if (!satisfies(version, '^2.0.2')) {
```

Comment:
changing the `satisfies` usage to `gte` should also work and avoid the regex

Reply:
That appears to be correct.

Reply:
PR updated.